charts/promtail/templates/service-metrics.yaml

```
@@ -7,8 +7,11 @@ metadata:
   labels:
     {{- include "promtail.labels" . | nindent 4 }}
     {{- toYaml .Values.service.labels | nindent 4 }}
+  {{- end }}
```

The additional labels should probably also be inside an `{{- if …}}`, or?

```suggestion
  labels:
    {{- include "promtail.labels" . | nindent 4 }}
    {{- with .Values.service.labels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```